entity-framework/core/querying/related-data.md

```diff
@@ -48,6 +48,10 @@ You can combine all of this to include related data from multiple levels and mul
 
 [!code-csharp[Main](../../../samples/core/Querying/Querying/RelatedData/Sample.cs#IncludeTree)]
 
+You may want to include multiple related entities for one of the entities that is being included. For example, when querying `Blog`s, you include `Posts` and then want to include both the `Author` and `Tags` of the `Posts`. To do this, you need to specify each include path starting at the root. For example, `Blog -> Posts -> Author` and `Blog -> Posts -> Tags`. This does not mean you will get redundant joins, in most cases EF will consolidate the joins when generating SQL.
```

Comment:
I'm sure it's possible to come up with one... just leaving room for the cases where you may get an extra join 😄